src/components/05_pages/NodeForm/index.js

```diff
@@ -77,10 +79,16 @@ class NodeForm extends React.Component {
           .includes(key),
       )
       .reduce((agg, [key, value]) => ({ ...agg, [key]: value }), {});
+
+    // Set default `Authored By` relationship.
+    if (!Object.prototype.hasOwnProperty.call(props, 'entity')) {
+      state.entity.data.relationships.uid.data = { ...props.user };
+    }
     return state;
   }
 
   componentDidMount() {
+    this.props.requestUser(1);
```

Comment:
Shouldn't this be the current user rather than user 1?

Reply:
It seems like you can just make a GET request to `/jsonapi/user` which returns the current user 👻

Reply:
Sorry, I overlooked that a bit since I wasn't logged in as uid 1. However, it seems like there's https://www.drupal.org/project/jsonapi/issues/2927037 which has been fixed in 2.x.

Reply:
Yeah. So we should probably consider upgrading to 2.x at some point.

Reply:
Here's a followup to upgrade to 2.x, https://github.com/jsdrupal/drupal-admin-ui/issues/380